### DISCUSSION TITLE
A better status showing - blocking - unblocking

### DISCUSSION THREAD

**Discussion by ninhnq** [AdguardTeam/AdGuardHome#3545] [Ideas]
Hi AdGuardTeam,
First of all, thank you for suck wonderful tool that you all put efforts into. I and my family certainly benefit a lot using it, so thank you!

I wonder if it is possible that we can have a better status showing - blocking - unblocking process for users.

My idea is as follow:
 
# **Status showing**
- **For now** When I go to a website that is showed unavailable, I cannot know if the address is invalid or the website is down or it is blocked by AdGuardHome, and if it is blocked, it fell into which category.
- **I would like** some kind of mechanism e.g. a notification page that show the status: Address invalid  /  It is down  / it is blocked by AdGuardHome for which reason.

# **Unblocking**
- **For now** After checking the admin page to see that the website is actually blocked. I click on the unblock button and wait for it to update so I can access the website.
- **I would like** a unblock button on the "notification page" mentioned above that do the blocking task and maybe notify me that how much time I have to wait and / or refresh the page automatically for time efficiency.

# **Blocking**
- **For now** When I go to a website and feel that it is generally bad and would like to block it so my kids and family wouldnot be annoyed by it, I have to go to the admin page and go through the process of blocking it.
- **I would like** some kind of tool e.g. browser extension that can do that blocking task faster and easier, and maybe share my opinion on the website to the world so other people may benefit from it and we can construct the blocking list collaboratively.

Thank you again and looking forward to your reply.
Regards,
Richard.

**Comment by ainar-g** [comment]
Hello and thanks for the ideas.  The first one you can kind of do already, using the “Custom IP” mode.  We'll document all ways of doing that in #3176.
